docs/explore/datadog/index-datadog.mdx

```diff
@@ -0,0 +1,484 @@
+---
+id: index-datadog
+title: Redis Enterprise Observability with DataDog
+sidebar_label: Redis Enterprise Observability with Datadog    
+slug: /explore/datadog
+authors: [ajeet,christian]
+---
+
+Devops and SRE practitioners are already keenly aware of the importance of system reliability, as it’s one of the shared goals in every high performing organization. Defining clear reliability targets based on solid data is crucial for productive collaboration between developers and SREs. This need spans the entire infrastructure from application to backend database services.
+
+Service Level Objectives (SLOs) provide a power interface for all teams to set clear performance and reliability goals based on Service Level Indicators (SLIs) or data points. A good model is to think of the SLIs as the data and the SLO as the information one uses to make critical decisions.
+
+Further Read: [https://cloud.google.com/blog/products/devops-sre/sre-fundamentals-slis-slas-and-slos](https://cloud.google.com/blog/products/devops-sre/sre-fundamentals-slis-slas-and-slos) 
+
+#### Redis
+
+
+Redis is a popular multi-model NoSQL database server that provides in-memory data access speeds for search, messaging, streaming, caching, and graph—amongst other capabilities. Highly performant sites such as Twitter, Snapchat, Freshworks, GitHub, Docker, Pinterest, and Stack Overflow all look to Redis to move data in real time.
+
+Redis SLOs can be broken down into three main categories:
+
+| Category   | Definition     | Example SLO       | Example SLI  |
+| ---------- | -------------- | ----------------- | ------------ |
+| Throughput | Number of operations being pushed through the service in a given time period | System should be capable of performing 200M  operations per second | redisenterprise.total_req |        
+| Latency    | Elapsed time it takes for an operation | Average write latency should not exceed 1 millisecond | redis_enterprise.avg_latency |
```

Comment:
The other two SLIs are `redisenterprise.`, this one is `redis_enterprise.` - is the `_` correct here, just asking as it seems inconsistent with the other two names, but may be how it is in the product, I don't know.

Reply:
Good catch. The metrics name starts with redisenterprise.... Fixed it.